### DISCUSSION TITLE
Longformer models for QA

### DISCUSSION THREAD

**Discussion by maylorian** [deepjavalibrary/djl#2450] [Q&A]
Hi!

Are longformer models currently supported for Question Answering tasks? I saw that models for BERT and RoBERTa exist but when loading a longformer based model, I always get the Model not found exception.

Thanks!

**Comment by frankfliu** [comment]
We didn't import this model in our model zoo. But you can trace it by yourself using our export script:
https://github.com/deepjavalibrary/djl/blob/master/extensions/tokenizers/src/main/python/model_zoo_importer.py

**Reply by maylorian** [reply]
Thanks! I did try that but when loading the model in DJL I get the error saying no model found with matching Input/Output  I assumed that it was due to the different inputs it required but if that's not the case, I need to revisit mt code.
